packages/server/__snapshots__/1_es_modules_spec.js

```diff
@@ -92,7 +92,7 @@ Module build failed (from [..]):
 SyntaxError: /foo/bar/.projects/e2e/lib/fail.js: Unexpected token (2:0)
 
   1 | export default {
-> 2 | 
+> 2 |
```

Comment:
I have no idea why this was necessary, but it kept failing without this snapshot update.